content/events/2023-kubecon-na.md

```diff
@@ -0,0 +1,12 @@
+title = "KubeCon CloudNativeCon NA 2023"
+date = "2023-09-05T17:00:00Z"
+description = "Join us as we all gain unparalleled insights into Kubernetes and cloud-native technologies. This will be a melting pot of information and ideas. Visit us at our booth so we can discuss how you can use Spin and Fermyon as part of your cutting edge ideas and solutions in the evolving cloud-native landscape."
```

Comment:
please ignore if this is a misreading of the text!
```suggestion
description = "Join us as we all gain unparalleled insights into Kubernetes and cloud-native technologies. This will be a melting pot of information and ideas. Visit us at our booth so we can discuss how you can use Spin and Fermyon Cloud as part of your cutting edge ideas and solutions in the evolving cloud-native landscape."
```